src/main/webapp/person/visualizePersonalInfo.jsp

```diff
@@ -64,7 +64,7 @@
 			<tr>
 				<td align="center">
 					<div class="items-container" data-toggle="tooltip" data-placement="right" title="${fr:message('resources.ApplicationResources', 'link.operator.submitPhoto')}">
-						<img src="${fr:checksum('/person/retrievePersonalPhoto.do?method=retrieveOwnPhoto')}" width="100" height="100" align="middle" style="border: 2px #eee solid"/>
+						<img src="${fr:checksum('/person/retrievePersonalPhoto.do?method=retrieveOwnPhoto&'.concat(person.personalPhotoEvenIfPending.externalId))}" width="100" height="100" align="middle" style="border: 2px #eee solid"/>
```

Comment:
If you need to pass the id then there is no point in having a method retrieveOwnPhoto. Probably there is a method by id that suits your need, and then delete the retrieveOwnPhoto, one less method to take care of.